openslides/agenda/static/templates/agenda/item-import.html

```diff
@@ -0,0 +1,86 @@
+ <h1 translate>Import agenda items</h1>
+
+<div id="submenu">
+  <a ui-sref="agenda.item.list" class="btn btn-sm btn-default">
+    <i class="fa fa-angle-double-left fa-lg"></i>
+    <translate>Back to overview</translate>
+  </a>
+</div>
+
+<h2 translate>Import by copy/paste</h2>
+<p translate>Copy and paste your agenda item titles in this textbox.
+Keep each item in a single line.</p>
```

Comment:
The newline between line 11 and line 12 will not be seen in the browser. Did you wan't a newline?